.github/workflows/codacy.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
This entire file looks like it was ripped off someone else's repo. It doesn't match upstream's template closely.